packages/eslint-config/index.js

```diff
@@ -157,6 +157,8 @@ export const config = [
           ],
         },
       ],
+      "turbo/no-undeclared-env-vars": "off",
+      "n/no-process-env": "off",
```

Comment:
The same here - we should have `on` (default) set - also not all apps / packages use `eslint/n` so it didn't make sense to disable this rule